### DISCUSSION TITLE
Not Able to Log In

### DISCUSSION THREAD

**Discussion by vksolanki1971** [rusq/slackdump#342] [General]
Hi All,

I have launched slackdump via Terminal and followed the prompts.  The browser window opens and I log in using proper credentials;  however, I recieved a message that the log in is not corect.  I feel like I'm doing something wrong.

Any ideas appreciated.

Thanks - V

**Comment by rusq** [comment]
Hey @vksolanki1971, when you're logging in can you confirm your workspace name is correct in the address bar?  If Slack in the browser says that your login details are incorrect, it means that they're incorrect ¯\_(ツ)_/¯ If it was a slackdump problem, you'd see an error in the terminal, but not in the browser.

**Comment by vksolanki1971** [comment]
Hi @rusq !

I'm pretty sure that I'm using the right workspace name creds - I used a different browser and was able to log in.  I'll try again.

V

**Comment by philr1985** [comment]
Hi there, I'm having a similar issue too. I run SlackDump get to the login page, when I login in I usually use the "Find My Workspace" method. I do this and login, but then seemingly nothing happens. 

Feel like I'm definitely doing something wrong! All help greatly appreciated!

**Reply by rusq** [reply]
Hey @philr1985, could you provide more details?

- what is the authentication method on the workspace?
- what were your menu choices if using wizard?
- what were the command line flags if you used CLI?
- Did you see any messages, and if yes, what were they?
- Were there any errors?

**Reply by philr1985** [reply]
Hi Rusq,

So I am using the Export option. 
I select the Login Browser option. 
Workspace name. 
Then I use Interactive to login. 
Obviously I put the workspace name in. 

I'm on an Intel MacBook Pro OS 13.7.1 

There were no errors, it just says to login as normal in the browser which I do. That's it, then there's nothing. No error messages or anything like that. I did try it on a Windows machine, and that did give an option for specifying the name of the file it's spits out at the end of the process. But similarly, I was having trouble on a Windows machine (different issues though.) 

TIA.

**Reply by rusq** [reply]
Hey, Phil, thanks for explaining.  What version of slackdump are you using?

**Reply by philr1985** [reply]
Version 3.0.0
<img width="567" alt="Screenshot 2024-11-21 at 16 39 52" src="https://github.com/user-attachments/assets/8d0ab8da-463e-4756-870a-d34dd34c71cf">

**Reply by rusq** [reply]
Got it, can you try "User Browser" during login?

1. First, do the cleanup:
   ```shell
   ./slackdump tools uninstall -legacy-browser
   ./slackdump tools uninstall
   ./slackdump workspace new <name of the your workspace or url>
   ```
   This will open the authentication menu.
2. Choose "User Browser"; it will show the supported browsers installed on you system, i.e. Edge and Google Chrome
3. Close that browser if it is open.
4. Press Enter in the slackdump - it will try and open the browser on the workspace you have entered on step 1.
5. Login into slack, if necessary.  If you're already logged in, it will capture the cookies and token and save them into the workspace credentials file
6. You should be authenticated.
7. Slackdump will select the new workspace automatically.
8. Run:
   ```
    slackdump wiz
    ```
   The main menu will open.

**Reply by philr1985** [reply]
Hi, 

Thanks for the reply. I’m so sorry but Step 1 where do I enter this? Just in a different instance of Terminal? I do a tiny bit of basic command line stuff but I’m not sure where I should do this? 

Phil

**Reply by philr1985** [reply]
I may do a quick screen record and upload it here maybe? So you can see what it is I'm doing and what I'm doing wrong! 🫣

**Reply by rusq** [reply]
> where do I enter this?

Ok, the easier way, I suppose, would be to use the UI.

Here are the UI steps to replace step 1 of the original instructions:

1. Main Menu->Tools->Uninstall
2. Run Uninstall
3. Exit to main menu again
4. Choose "Workspace"->"New" -> "Login in Browser"
5. Enter slack workspace name

After these steps, continue from step 2 of the original instructions.

**Reply by philr1985** [reply]
Hi Rusq, 

I really want to get this to work, and I haven't bought you a coffee or a tea yet, but will once I figure out what I'm doing. I appreciate your patience! 

I'm just really struggling with these instructions. I don't see any "Main Menu" "Tools" or "Uninstall" options. All I see is the screenshot I posted above. Which has 3 options, Print help and exit, Run Wizard and Exit. 

Do you have an e-mail address where we can converse, privately because then I can send you more screenshots etc. Perhaps once I figure out what's going on, I can post the solution here, or you can. 

Thanks Rusq in advance.

**Reply by rusq** [reply]
Hey, no need for coffee, you can join the telegram group: https://t.me/slackdump

On that menu from the screenshot, choose "Run wizard", and you'll be in the main menu.

**Comment by vksolanki1971** [comment]
I figured this out! I logged out of slack on a normal browser then clicked
to log back in. The URL after the domain is different from what the
Slackdump browser navigates to. Using that URL works.

**Reply by rusq** [reply]
Hey @vksolanki1971 , great that you figured it out.

Could you describe, using the "acme" as the name of the company, what was the URL provided by slackdump and what was the actual URL?  I'll see if this is something that can be patched for the benefit of others

**Comment by vksolanki1971** [comment]
Hi!

What I did was used the following after the domain:
?redir=%2Fgantry%2Fclient

The acme URL is:
https://acme.slack.com/?redir=%2Fgantry%2Fclient

Hth - V

**Reply by rusq** [reply]
Thanks a lot! I'll experiment with this.
